src/librustc-builtins/builtins

```diff
@@ -0,0 +1 @@
+../compiler-rt/lib/builtins
```

Comment:
I think this may be a symlink? We may want to remove this for now and just assume the location of `lib/builtins` is what we expect